### PR TITLE
fix: tabindex should return if greater than 0

### PR DESCRIPTION
**Pull request by michael-siek**

### CODE REVIEW

**Comment by KamiKillertO** [comment]
@michael-siek the purpose of this rule is to ensure tags have a positive tabIndex. So the rule should report only when the tabIndex is negative.

**Comment by michael-siek** [comment]
@KamiKillertO Based on what w3c states tabindex should not be positive https://www.w3.org/TR/WCAG20-TECHS/F44.html
https://www.w3.org/wiki/HTML/Usage/tabindex

**Comment by KamiKillertO** [comment]
🤔 ok, I think I was not clear enough 😟. I don't mind making this change. I agree that tabIndex should not be higher than 0. But, the rule documentation clearly stated that `all focusable elements must have a positive tabindex attribute`. So, we need to update the rule documentation and the error [message](https://github.com/linthtml/linthtml/blob/develop/lib/messages.js#L36) to indicate that tabindex should be equal to  `-1` or `0`.

**Comment by michael-siek** [comment]
@KamiKillertO I've updated the docs and also fixed all the unit test to go along with the changes required

File: lib/rules/focusable-tabindex-style/index.js [hunk 1]
```diff
@@ -13,14 +13,22 @@ function isLinkable(node) {
...
+    isFocusableNatively || isLinkable(node) || hasNonEmptyAttr(node, "tabindex")
+  );
 }
 
 function isDisabled(element) {
   return !!element.attribs && !!element.attribs.disabled;
 }
 
+function isNonInteractive(node) {
+  return ["div"].indexOf(node.name) !== -1;
```
[the remark this answers is not in the document]
Comment (michael-siek) [reply lib/rules/focusable-tabindex-style/index.js]: Changes made

**Comment by KamiKillertO** [comment]
@michael-siek I've released the version 0.4.0-beta.3 which contain this change :+1:

**Pull request merged**